packages/plugins/plugin-adjust/src/AdjustPlugin.tsx

```diff
@@ -35,6 +37,24 @@ export class AdjustPlugin extends DestinationPlugin {
 
     const adjustConfig = new AdjustConfig(this.settings.appToken, environment);
 
+    if (this.adjust !== undefined) {
+      adjustConfig.setAttributionCallbackListener(function (attribution) {
```

Comment:
oh, it's the first time I see this notation for an inline function, TIL!

Reply:
Oh, I see why you created the `const analytics`, so that you captured the value of the analytics in the clousure.
You can preserve the `this` reference if you declare your function here with an arrow:
```
adjustConfig.setAttributionCallbackListener((attribution) => {
	...
	this.analytics?.track('Install Attributed', trackPayload); // Arrow functions closure on this automatically
});

Reply:
ohhhh that is way simpler!!